spec/controllers/orders_controller_spec.rb

```diff
@@ -1,6 +1,37 @@
 require "rails_helper"
 
 RSpec.describe OrdersController, type: :controller do
+  describe "GET #index" do
+    subject { get :index }
+
+    context "user has already signed in" do
+      login
+
+      let(:order_1) { create(:order, user: @user, created_at: 3.weeks.ago) }
+      let(:order_2) { create(:order, user: @user, created_at: 2.weeks.ago) }
+      let(:order_3) { create(:order) }
+      let(:orders) { [order_2, order_1] }
+
+      before do
+        subject
+      end
+
+      it "is successful and creates new order with some order items based on the ticket types of this event" do
+        expect(response).to be_success
+        expect(assigns(:orders)).to eq orders
+      end
+    end
+
+    context "user has not signed in yet" do
+      before do
+        expect(controller).to receive(:store_location_and_require_login).and_call_original
```

Comment:
We shouldn't need this anymore since the logic is tested in "require signing in"